tests/ceph_ansible/test_ansible_roll_over.py

```diff
@@ -129,10 +129,27 @@ def run(ceph_cluster, **kw):
 
     log.info("Ceph versions " + ceph_installer.get_installed_ceph_versions())
 
+    # Run site-x.yml by default
+    if ceph_cluster.ansible_config.get('containerized_deployment'):
```

Comment:
Is site-container.yaml will be considered here?

Reply:
Thanks for the heads-up
Changed accordingly